src/features/thai/thaiLigatures.js

```diff
@@ -0,0 +1,40 @@
+/**
+ * Apply Thai Ligatures feature to tokens
+ */
+
+import { ContextParams } from '../../tokenizer';
```

Comment:
I just noticed that the file extensions are missing, but are now required since we changed the build process and the library may be used in module form on the web, where we need the file extensions. I'm fixing it right now.

Reply:
@rafallyczkowskiadylic Would you mind allowing edits to the PR by maintainers, please?

Reply:
Sure, added. Believe covered all